src/index.html

```diff
@@ -54,42 +55,124 @@
 				Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
 		</div>
 	</div>
 
+	<nav class="navbar navbar-static bar-wrapper navbar-expand-lg .custom-carets ">
 
-	<section class="image-wrapper">
-		<div class="container">
-			<div class="row">
-				<div class="col-md-4 col-sm-6 image-hover mx-auto">
-					<a href="#">
-						<img src="res_d7d63673fac35dac5f93604c4a7bd059_450x450_5t3l.jpg" class="img image-style">
+		<div class="container nav-div">
+			<ul class="navbar-nav ">
+				<li class="nav-item dropdown">
+					<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
+					    aria-expanded="false">
+						CATEGORY
 					</a>
-				</div>
-				<div class="col-md-4 col-sm-6 mx-auto image-hover">
-					<a href="#">
-						<img src="res_d7d63673fac35dac5f93604c4a7bd059_450x450_5t3l.jpg" class="img image-style">
+					<div class="dropdown-menu" aria-labelledby="navbarDropdown">
+						<a class="dropdown-item" href="#">Action</a>
+						<a class="dropdown-item" href="#">Another action</a>
+						<div class="dropdown-divider"></div>
+						<a class="dropdown-item" href="#">Something else here</a>
+					</div>
+				</li>
+				<li class="nav-item dropdown">
+					<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
+					    aria-expanded="false">
+						SORTING
 					</a>
-				</div>
-				<div class="col-md-4 col-sm-6 mx-auto image-hover">
-					<a href="#">
-						<img src="res_d7d63673fac35dac5f93604c4a7bd059_450x450_5t3l.jpg" class="img image-style">
+					<span class="caret"></span>
+					<div class="dropdown-menu" aria-labelledby="navbarDropdown">
+						<a class="dropdown-item" href="#">Action</a>
+						<a class="dropdown-item" href="#">Another action</a>
+						<div class="dropdown-divider"></div>
+						<a class="dropdown-item" href="#">Something else here</a>
+					</div>
+				</li>
+				<li class="nav-item dropdown">
+					<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
+					    aria-expanded="false">
+						VEGETARIAN
 					</a>
-				</div>
+					<div class="dropdown-menu" aria-labelledby="navbarDropdown">
+						<a class="dropdown-item" href="#">Action</a>
+						<a class="dropdown-item" href="#">Another action</a>
+						<div class="dropdown-divider"></div>
+						<a class="dropdown-item" href="#">Something else here</a>
+					</div>
+				</li>
+			</ul>
+
+			<div class="slider-nav ">
+				<p id="nav-price">PRICE</p>
+				<section class="demo">
+					 <input type="range" multiple value="0,100" />
+				</section>
 			</div>
 
+
+
+
 		</div>
-	</section>
+	</nav>
+
+
+	<div class="row" id="wrapper"></div>
+	<div class="container">
+
```

Comment:
Remove extra lines.

I would suggest adding a linter in your editor and fix all the warnings